bricoler

```diff
@@ -2888,7 +2888,7 @@ the VM is shut down.
                 "-j", self.parallelism or self.ncpus,
                 "-o", reportpath,
                 "-r", dbpath,
-                self.tests,
+                "\"" .. self.tests .. "\"",
```

Comment:
Because this is a positional parameter, I think it'd be a bit neater to make the helper script handle it. That is, in `run-kyua`, can we change `TEST_PATH=$1` to `TEST_PATH=$@` to fix the problem?

I'm a bit buried this morning so can't test right away -- if the suggestion works and you have time to verify it, I'd prefer that approach. Otherwise I'll come back to it this afternoon.